docs/java_cli_for_managedgateway.rst

```diff
@@ -156,20 +156,20 @@ Refer to the `documentation <https://docs.internetofthings.ibmcloud.com/devices/
 Unmanage request - gateway
 -----------------------------------------------------
 
-A gateway can invoke sendGatewayUnmanageRequet() method when it no longer needs to be managed. The IBM Watson Internet of Things Platform will no longer send new device management requests for this gateway and all device management requests from the gateway (only for the gateway and not for the attached devices) will be rejected other than a **Manage** request.
```

Comment:
Yeah, its a typo, unfortunately in the library code itself the method is created as "sendGatewayUnmanageRequet". So for now please use the same name, and once we update the library we can modify with proper name.